src/components/totals/totals.ts

```diff
@@ -0,0 +1,38 @@
+import { Component } from '@angular/core';
+import { Observable } from 'rxjs/Observable';
+import { select, NgRedux } from '@angular-redux/store';
+import { FaultStoreProvider } from '../../providers/fault-store/fault-store';
+import { IFaultState, IFaultElementState } from '../../providers/fault-store/fault-store.model';
+
+interface IFaultCounter {
+  faults: IFaultElementState,
+  totals: IFaultState
+}
+
+/**
+ * Generated class for the TotalsComponent component.
+ *
+ * See https://angular.io/api/core/Component for more info on Angular
+ * Components.
+ */
+@Component({
+  selector: 'totals',
+  templateUrl: 'totals.html'
+})
+export class TotalsComponent {
+
+  @select(['totals', 'fault']) readonly faultMarks$: Observable<number>;
```

Comment:
Are we using $ to indicate an observable?

Reply:
yes, It seems to be best practice in all the examples and docs I've read